From bdfef068c225406219cee665c8aa086cf8a98a0c Mon Sep 17 00:00:00 2001
From: Andreas Bauer <a10r@users.noreply.github.com>
Date: Wed, 26 Jun 2024 13:54:18 +0200
Subject: [PATCH] mpv: disable lua dependency

---
 packages/mpv.cmake | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index 9cb510e..c581df7 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -11,7 +11,6 @@ ExternalProject_Add(mpv
         libiconv
         libjpeg
         libpng
-        luajit
         rubberband
         uchardet
         openal-soft
@@ -39,7 +38,7 @@ ExternalProject_Add(mpv
         ${mpv_lto_mode}
         -Dlibmpv=true
         -Dpdf-build=enabled
-        -Dlua=enabled
+        -Dlua=disabled
         -Djavascript=enabled
         -Dsdl2=enabled
         -Dlibarchive=enabled
-- 
2.35.2.windows.1

